src/imageLoader/wadouri/dataSetCacheManager.js

```diff
@@ -28,8 +27,7 @@ function get (uri) {
 
 // loads the dicom dataset from the wadouri sp
 function load (uri, loadRequest = xhrRequest, imageId) {
-  const cornerstone = external.cornerstone;
-
+  const { cornerstone, dicomParser } = external;
```

Comment:
Ooh. I actually think this is cleaner than what we had, but why do it here instead of at the top with the rest of the imports?